RELEASE_NOTES.md

```diff
@@ -4,6 +4,13 @@
 - [KOGITO-2905](https://issues.redhat.com/browse/KOGITO-2905) - Change in image definition in the CRDs
 - [KOGITO-3039](https://issues.redhat.com/browse/KOGITO-3039) - Agnostic infrastructure layer configuration on Operator
 - [KOGITO-2113](https://issues.redhat.com/browse/KOGITO-2113) - Simplify binary build deployment
+
+
 ## Bug Fixes
 
 - [KOGITO-3382](https://issues.redhat.com/browse/KOGITO-3382) - Transitive dependency of Keycloak operator module missing
+
+
+## Known Issues
+- [PLANNER-2084](https://issues.redhat.com/browse/PLANNER-2084) - Building image from Quarkus/Optaplanner project
+    - [KOGITO-3433](https://issues.redhat.com/browse/KOGITO-3433) - Update Graalvm version to 20.2.0
```

Comment:
indentation ?

Reply:
I think _Known Issues_  deserves a more verbose approach. What about:

> OptaPlanner applications are not compatible with native builds duo to an incompatibility with Graalvm version 20.2.0 included in Kogito Images. Please see  [PLANNER-2084](https://issues.redhat.com/browse/PLANNER-2084) for more details.

Reply:
@radtriste not really, I put the kogito one under the optaplanner issue as they are related.
but makes more sense describe it better as @ricardozanini suggested.